patina_dxe_core/examples/std.rs

```diff
@@ -168,7 +170,9 @@ fn build_hob_list() -> *const c_void {
             },
             owner: efi::Guid::from_fields(0, 0, 0, 0, 0, &[0u8; 6]),
             resource_type: hob::EFI_RESOURCE_FIRMWARE_DEVICE,
-            resource_attribute: hob::EFI_RESOURCE_ATTRIBUTE_PRESENT | hob::EFI_RESOURCE_ATTRIBUTE_INITIALIZED,
+            resource_attribute: hob::EFI_RESOURCE_ATTRIBUTE_PRESENT
+                | hob::EFI_RESOURCE_ATTRIBUTE_INITIALIZED
+                | hob::EFI_RESOURCE_ATTRIBUTE_WRITE_BACK_CACHEABLE,
```

Comment:
If this is a memory-mapped firmware device, I think uncacheable would provide a better example.